assets/src/scripts/store/index.js

```diff
@@ -345,6 +344,12 @@ export const Actions = {
             upstreamBasin
         };
     },
+    setNetworkFeatures(networkSites) {
```

Comment:
Please remove this as it does not belong here.

Reply:
Removed setNetworkFeatures from main store.